pkg/cmd/roachtest/tests/replicagc.go

```diff
@@ -62,7 +62,10 @@ func runReplicaGCChangedPeers(
 
 	c.Put(ctx, t.Cockroach(), "./cockroach")
 	c.Put(ctx, t.DeprecatedWorkload(), "./workload", c.Node(1))
-	settings := install.MakeClusterSettings(install.EnvOption([]string{"COCKROACH_SCAN_MAX_IDLE_TIME=5ms"}))
+	settings := install.MakeClusterSettings(install.EnvOption([]string{
+		"COCKROACH_SCAN_MAX_IDLE_TIME=5ms",
```

Comment:
drive-by:

`COCKROACH_SCAN_MAX_IDLE_TIME` "should" imply a ScanInterval of `num_replicas* max_idle_time * avg_processing_duration`, shouldn't it? At least that's what the comment[^1] suggests. So is this not working as advertised?

[^1]: https://github.com/cockroachdb/cockroach/blob/58004827905fbec8926a51ea6f751823b1991fa4/pkg/server/config.go#L415-L419

Reply:
You are right, I'm still running tests locally to see how behaviour changes. Created a PR speculatively.
Something is preventing any moves from happening for 5 minutes so it seems at least now that I added logging on each replica count check attempt.

Reply:
Funny thing... `COCKROACH_SCAN_MAX_IDLE_TIME` is only applied to first 3 nodes, but nodes doing rebalancing are the second set of nodes which start without any options. So maybe that's the cause.